docs/source/nodes_and_pipelines/namespaces.md

```diff
@@ -0,0 +1,336 @@
+# Reuse pipelines with namespaces
+
+## How to reuse your pipelines
+
+If you want to create a new pipeline that performs similar tasks with different inputs/outputs/parameters as your `existing_pipeline`, you can use the same `pipeline()` creation function as described in [How to structure your pipeline creation](modular_pipelines.md#how-to-structure-your-pipeline-creation). This function allows you to overwrite inputs, outputs, and parameters. Your new pipeline creation code should look like this:
+
+```python
+def create_new_pipeline(**kwargs) -> Pipeline:
+    return pipeline(
+    [existing_pipeline], # Name of the existing pipeline
```

Comment:
Why is this`[existing_pipeline]` instead of  `existing_pipeline`? Also the comment is a bit inaccurate: it's not `# Name of the existing pipeline` but rather a `Pipeline` object itself.

Reply:
I think the example would be more realistic if `existing_pipeline` had a simple but realistic job and clear name. E.g. like `cook_pipeline` in old example.

Reply:
I fixed the comment and removed the brackets, but I believe `existing_pipeline` is a sufficiently descriptive name to define reusability. A more detailed example is provided a bit further down, based on the spaceflights starter.